instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/EnableOpenTelemetryTracing.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Auto-configures OpenTelemetry instrumentation
+ *
+ * <p>Enables OpenTelemetry Tracing in spring applications by completing a component scan
+ *
+ * <p>
```

Comment:
Something is off-with the javadoc. You're starting an empty paragraph, perhaps instead of </p> closing the last one.

Reply:
it doesn't feel like there is a use of breaking this into 2 lines.  Can it simply be:

```
Auto-configures OpenTelemetry. Enables Open Telementry Tracing in Spring applications by 
completing a component scan
```

Reply:
Fixed